src/instrumenter/transpiling_context.ts

```diff
@@ -194,35 +192,40 @@ export class TranspilingContext {
             bindingsVarType
         );
 
-        if (gte(instrCtx.compilerVersion, "0.8.0")) {
-            const containerBody = this.containerFun.vBody as Block;
-            if (
-                this.instrSiteType === InstrumentationSiteType.FunctionAnnotation ||
-                this.instrSiteType === InstrumentationSiteType.StateVarUpdated
-            ) {
-                const uncheckedBlock = this.factory.makeUncheckedBlock([]);
-                this.uncheckedBlocks.push(uncheckedBlock);
-                containerBody.insertAtBeginning(uncheckedBlock);
-
-                this.oldMarkerStack = [[uncheckedBlock, "end"]];
+        /// TODO(dimo): This logic is convoluted and doesn't belong in the constructor. Instead perhaps we should have
```

Comment:
Address TODO

Reply:
Fixed in [a482f6b](https://github.com/ConsenSys/scribble/pull/154/commits/a482f6b39338dedcd43bdfe3f23939aed396ad48).